content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,145 @@
-===================================
-Connect an IoT Box to your database
-===================================
+============================================================
+Connect an Internet of Things (IoT) box to the Odoo database
+============================================================
 
-Install the Internet of Things (IoT) App on your Odoo Database.
+The Internet of Things (IoT) box is a micro-computer device that allows for the connection of
+devices to a Odoo database. A subscription is required in order to use the
```

Comment:
"A subscription is required..."
a bit ambigious in my opinion, do you mean the "IoT box subscription" or the Odoo subscription ?

Reply:
Great suggestion. I've added that it needs to be an IoT box subscription. 

And I've added this block in: 
.. seealso::
   `IoT Box FAQ <https://www.odoo.com/app/iot-faq>`__